ProjectTemplates/DotNetTemplate/MonoGame.Templates.FSharp/content/MonoGame.Application.DesktopGL.FSharp/Game1.fs

```diff
@@ -0,0 +1,49 @@
+namespace MGNamespace
+
+open Microsoft.Xna.Framework
+open Microsoft.Xna.Framework.Graphics
+open Microsoft.Xna.Framework.Input
+
+type Game1 () as this =
+    inherit Game()
+ 
+    let mutable graphics = Unchecked.defaultof<GraphicsDeviceManager>
```

Comment:
I dont think you need to include the generic types here as they will be inferred.

Reply:
Not sure what change you want me to make with this?

Reply:
It could be inferred with:
```
let mutable graphics = Unchecked.defaultof<_>
```
Its also ok to leave it though.